lib/checkout.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'order'
+require_relative 'till'
+
+class Checkout
+
+  include Till
```

Comment:
as we discussed, modules usually have the names of abilities. So "Tilllike", weird though it sounds, might be appropriate here.